### PR TITLE
Updated OCP on RHV team members

### PR DESCRIPTION
**Pull request by janosdebugs**
This PR updates the OCP on RHV team members to reflect the new team structure.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,9 +1,11 @@
 approvers:
-  - bennyz
-  - bertinatto
```
Comment (dobsonj) [inline OWNERS:3]: @bertinatto do you want to stay on the approver list?

**Comment by dobsonj** [comment]
/retest
can't fail due to OWNERS file

File: OWNERS (hunk 1 above)
Comment (bertinatto) [reply OWNERS:3]: I'm OK being dropped.

**Pull request merged**